transformer_engine/jax/cpp_extensions/attention.py

```diff
@@ -1356,6 +1384,7 @@ def fused_attn_fwd(
         dropout_probability=dropout_probability,
         is_training=is_training,
         max_segments_per_seq=max_segments_per_seq,
+        window_size=window_size,
```

Comment:
Handle it before going to the low level API.
```suggestion
        window_size=(-1, -1) if window_size is None else window_size,
```

Reply:
Revised as suggested